### ISSUE TITLE
[Bug]: Website: yarn fails due to kubectl version > 1.23

### ISSUE THREAD

**Issue by oktab1** [aws-samples/eks-workshop-v2#548]
### Installation method

Own AWS account

### What happened?

OS: MacOS Apple M1
IDE: Visual Studio Code
Python Version: 3.10.7
Yarn Version: 1.22.19
Node Version: v20.2.0

When running the website locally according to [documentation](https://github.com/aws-samples/eks-workshop-v2/blob/main/website/README.md), `yarn start` and `yarn build` fail because kubectl version is greater than `1.23`. The following errors appear in the deployed webpage `http://localhost:3000` and terminal.

**VSCode Terminal (__multiple similar errors__)**
```
Error running shell command: Error: Command failed: kubectl kustomize /Users/tolub/Documents/DevEnv/eks-workshop-v2/environment/workspace/modules/introduction/kustomize
error: invalid Kustomization: json: cannot unmarshal string into Go struct field Kustomization.patches of type types.Patch
```

**Webpage**
![image](https://github.com/aws-samples/eks-workshop-v2/assets/4214903/14d7591a-520d-4c91-9e81-d892716e2eaa)

### What did you expect to happen?

`yarn build`
Successful creation of `build` folder

`yarn start`
Successful launch of local eks-workshop website in default browser at `http://localhost:3000` without errors.

### How can we reproduce it?

* Install `kubectl` version greater than `1.23`
* Run `yarn build` or `yarn start` from `eks-workshop/website` folder

### Anything else we need to know?

On MacOS M1, it is a lot easier to install `kubectl-1.23` using [MacPorts](https://ports.macports.org/port/kubectl-1.23/) compared to other methods.

### EKS version

EKS version 1.23

**Comment by mylesw42** [comment]
An alternate work around to installing kubectl 1.23 is to download the release directly and put it in your $PATH. For me this was in $HOME/bin.

Double check what build you need in the URL. ie. darwin/linux, amd64, arm64

https://storage.googleapis.com/kubernetes-release/release/v1.23.0/bin/darwin/amd64/kubectl

**Comment by niallthomson** [comment]
This was resolved by #739
